CalculatorTests/CalculatorBrainTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by Wayne Rumble on 05/06/2017.
+//  Copyright © 2017 Wayne Rumble. All rights reserved.
+//
+
+import XCTest
+@testable import Calculator
+
+class CalculatorBrainTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testConstants() {
+        let symbolsAndValues = ["π": Double.pi, "e": M_E]
```

Comment:
you could split this into multiple tests with names that describe each one. If the implementation was changed such that one of these broke, would the test failure message be clear as to which broke or would it require investigation?

```
let brain = CalculatorBrain()

func testBrainEvaluatesPiOperandAsPiToSixDecimalPlaces() {
    brain.performOperation("π");

    XCTAssertEqual(brain.description!, Double.pi.roundTo(places: 6));
}
```